pgmpy/inference/ApproxInference.py

```diff
@@ -258,10 +256,16 @@ def map_query(
         --------
         >>> from pgmpy.utils import get_example_model
         >>> from pgmpy.inference import ApproxInference
+        >>> from pgmpy.factors.discrete import State,TabularCPD
         >>> model = get_example_model("alarm")
         >>> infer = ApproxInference(model)
         >>> print(infer.map_query(variables=["HISTORY", "CVP"]))
         {'HISTORY': 'FALSE', 'CVP': 'NORMAL'}
+        >>> virtual_evidence_history = TabularCPD(variable='HISTORY', variable_card=2,values=[[0.99],[0.01]],
+                                          state_names={"HISTORY": ["TRUE", "FALSE"]})
```

Comment:
@Nimish-4 Line continuation in docstring need `...`. Example: https://github.com/pgmpy/pgmpy/blob/dev/pgmpy/models/BayesianNetwork.py#L151

Reply:
Done.